diff --git a/GitPractice/trunk-master/sources/Utils.cpp b/GitPractice/trunk-master/sources/Utils.cpp
index 13a00b9..9cbab4c 100644
--- a/GitPractice/trunk-master/sources/Utils.cpp
+++ b/GitPractice/trunk-master/sources/Utils.cpp
@@ -39,4 +39,8 @@ int Utils::Random(int min, int max)
 {
 	int random = rand() % (max - min + 1) + min;
 	return random;
+}
+int Add(int a, int b)
+{
+return a + b;
 }
\ No newline at end of file
